env/AWSPW.env

```diff
@@ -43,6 +43,17 @@ if [[ "${step}" = "fcst" ]] || [[ "${step}" = "efcs" ]]; then
     export APRUN_UFS="${launcher} -n ${ufs_ntasks}"
     unset nnodes ufs_ntasks
 
+elif [[ "${step}" = "prep_emissions" ]]; then
+
+    export APRUN="${APRUN}"
+
```

Comment:
If `APRUN` is already exported, this achieves nothing and is unnecessary.

Reply:
That is true.
Paragraph removed.

Reply:
But need to export "APRUN" above.

Reply:
`APRUN` is constructed with `launcher`, so you should probably look to use that.

Reply:
Yes, APRUN is constructed in an earlier block, as:

APRUN="${launcher} -n ${ntasks}"

I just modified it to:

export APRUN="${launcher} -n ${ntasks}"

Without "export" will cause a configuration error.

Lines:
elif [[ "${step}" = "prep_emissions" ]]; then

    export APRUN="${APRUN}"
   
was copied from "HERA.env".